### PR TITLE
2.4.0

### LINKED ISSUE #744: 2.4.0 updates
- [x]  `orderBy` updates
- [x] `sqlite://` prefix updates
- [x] `$` prefixing

### PR DESCRIPTION
**Pull request by mhwelander**
* Sort by multiple
* Updated client-level methods to `$`
* Mentioned introspection respecting column order

### CODE REVIEW

File: content/03-reference/01-tools-and-interfaces/02-prisma-client/090-raw-database-access.mdx [hunk 1]
```diff
@@ -20,22 +20,22 @@ Use cases for raw SQL include:
 
 ## <inlinecode>queryRaw</inlinecode>
 
-`queryRaw` returns actual database records. For example, the following `SELECT` query returns all fields for each record in the `User` table:
+`$queryRaw` returns actual database records. For example, the following `SELECT` query returns all fields for each record in the `User` table:
 
 ```ts no-lines
 const result = await prisma.queryRaw('SELECT * FROM User;')
-```
+````
```
Comment (Jolg42) [inline content/03-reference/01-tools-and-interfaces/02-prisma-client/090-raw-database-access.mdx]: 4 backticks?

File: content/03-reference/01-tools-and-interfaces/02-prisma-client/100-connection-management.mdx [hunk 2]
```diff
@@ -34,25 +34,25 @@ main()
...
-The `disconnect` method closes the database connections that were established when `connect` was called and stops the process that was running Prisma's query engine.
+The `$disconnect` method closes the database connections that were established when`$connect` was called and stops the process that was running Prisma's query engine.
 
-Note that `connect` returns a `Promise`, so you should call it inside an `async` function with the `await` keyword.
+Note that`$connect` returns a `Promise`, so you should call it inside an `async` function with the `await` keyword.
 
 ## Connection pool
 
-Once `connect` was called, the query engine immediately creates a connection pool with the amount of connections that were specified as the `connection_limit` parameter on your database connection URL.
+Once`$connect` was called, the query engine immediately creates a connection pool with the amount of connections that were specified as the`$connection_limit` parameter on your database connection URL.
```
Comment (Jolg42) [inline content/03-reference/01-tools-and-interfaces/02-prisma-client/100-connection-management.mdx]:
```suggestion
Once`$connect` was called, the query engine immediately creates a connection pool with the amount of connections that were specified as the`connection_limit` parameter on your database connection URL.
```

File: content/03-reference/01-tools-and-interfaces/02-prisma-client/100-connection-management.mdx [hunk 3]
```diff
@@ -63,7 +63,7 @@ datasource postgresql {
 }
 ```
 
-If the `connection_limit` argument is omitted, the default number of connections is calculated according to this formula: `num_physical_cpus * 2 + 1` where `num_physical_cpus` represents the number of physical CPUs on your machine.
+If the`$connection_limit` argument is omitted, the default number of connections is calculated according to this formula: `num_physical_cpus * 2 + 1` where `num_physical_cpus` represents the number of physical CPUs on your machine.
```
Comment (Jolg42) [inline content/03-reference/01-tools-and-interfaces/02-prisma-client/100-connection-management.mdx]:
```suggestion
If the`connection_limit` argument is omitted, the default number of connections is calculated according to this formula: `num_physical_cpus * 2 + 1` where `num_physical_cpus` represents the number of physical CPUs on your machine.
```

File: content/04-guides/01-database-workflows/06-cascading-deletes/01-postgresql.mdx [hunk 4]
```diff
@@ -532,7 +532,7 @@ const { PrismaClient } = require('@prisma/client');
 const prisma = new PrismaClient();
 
 async function main() {
-  const userWithPost = await prisma.oneMoreUser.create({
+  const userWithPost = await prisma.$oneMoreUser.create({
```
Comment (Jolg42) [inline content/04-guides/01-database-workflows/06-cascading-deletes/01-postgresql.mdx]: Why is there a $ here? 🤔

Comment (Jolg42) [reply content/04-guides/01-database-workflows/06-cascading-deletes/01-postgresql.mdx]: and below too (same thing)

File: content/03-reference/01-tools-and-interfaces/02-prisma-client/090-raw-database-access.mdx (hunk 1 above)
Comment (mhwelander) [inline content/03-reference/01-tools-and-interfaces/02-prisma-client/090-raw-database-access.mdx]:
```suggestion
```
```

File: content/04-guides/01-database-workflows/06-cascading-deletes/01-postgresql.mdx (hunk 4 above)
Comment (mhwelander) [reply content/04-guides/01-database-workflows/06-cascading-deletes/01-postgresql.mdx]: Urgh - I searched for `.on. Good catch!

Comment (mhwelander) [inline content/04-guides/01-database-workflows/06-cascading-deletes/01-postgresql.mdx]:
```suggestion
  const userWithPost = await prisma.oneMoreUser.create({
```

Comment (Jolg42) [reply content/04-guides/01-database-workflows/06-cascading-deletes/01-postgresql.mdx]: Classic search and replace 😄 Happens to everyone once in a while 💥

File: content/03-reference/01-tools-and-interfaces/02-prisma-client/090-raw-database-access.mdx [hunk 5]
```diff
@@ -20,22 +20,21 @@ Use cases for raw SQL include:
 
 ## <inlinecode>queryRaw</inlinecode>
 
-`queryRaw` returns actual database records. For example, the following `SELECT` query returns all fields for each record in the `User` table:
+`$queryRaw` returns actual database records. For example, the following `SELECT` query returns all fields for each record in the `User` table:
 
 ```ts no-lines
 const result = await prisma.queryRaw('SELECT * FROM User;')
-```
```
Comment (Jolg42) [inline content/03-reference/01-tools-and-interfaces/02-prisma-client/090-raw-database-access.mdx:27]: This removal breaks the page
https://5f296ef0f33d09000717f30d--prisma2.netlify.app/reference/tools-and-interfaces/prisma-client/raw-database-access#inlinecodequeryrawinlinecode
Screen Shot 2020-08-04 at 16 26 28

**Pull request merged**